ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/show/properties/ShowTablePropertiesOperation.java

```diff
@@ -67,8 +69,10 @@ public int execute() throws HiveException {
       } else {
         Map<String, String> properties = new TreeMap<String, String>(tbl.getParameters());
         for (Entry<String, String> entry : properties.entrySet()) {
-          ShowUtils.appendNonNull(builder, entry.getKey(), true);
-          ShowUtils.appendNonNull(builder, entry.getValue());
+          if (!entry.getKey().equals(TABLE_IS_CTLT)) {
```

Comment:
Why `TABLE_IS_CTAS` is not checked here?

Reply:
Good question. I think it was just an omission. I'll add that in here too